modules/jaeger-config-storage.adoc

```diff
@@ -157,6 +118,152 @@ Minimum deployment = 16Gi*
 |`ZeroRedundancy`(no replica shards), `SingleRedundancy`(one replica shard), `MultipleRedundancy`(each index is spread over half of the Data nodes), `FullRedundancy` (each index is fully replicated on every Data node in the cluster).
 |
 
+4+|Elasticsearch index and index cleaner configuration options
+
```

Comment:
Index cleaner is relevant to auto and self provisioned elasticsearch, so should be moved into a common section.

Reply:
But wouldn't you specify this either under  "storage/elasticsearch" or "storage/options/es"?  That was why I had it in both tables.  I'm worried that if I put in under a common section that it might be confusing where to put in the YAML.

Reply:
No, the index cleaner configuration is directly under the `storage` node: https://www.jaegertracing.io/docs/1.20/operator/#elasticsearch-index-cleaner-job